packages/x-date-pickers/src/themeAugmentation/props.d.ts

```diff
@@ -35,27 +35,27 @@ import {
 
 export interface PickersComponentsPropsList {
   MuiCalendarOrClockPicker: CalendarOrClockPickerProps<unknown, unknown>;
-  MuiCalendarPicker: CalendarPickerProps<unknown>;
-  MuiCalendarPickerSkeleton: CalendarPickerSkeletonProps;
   MuiClock: ClockProps<unknown>;
   MuiClockNumber: ClockNumberProps;
   MuiClockPicker: ClockPickerProps<unknown>;
   MuiClockPointer: ClockPointerProps;
   MuiDateField: DateFieldProps<unknown, unknown>;
+  MuiDateCalendar: DateCalendarProps<unknown>;
```

Comment:
Nitpick: If we were to follow alphabetical order, this one should change places with `MuiDateField` and `MuiDayCalendarSkeleton` would end up below `MuiDayCalendar` 🙈